### PR TITLE
Mes 4.12.1 analytics

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Add analytic changes
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/test-report/__tests__/test-report.analytics.effects.spec.ts [hunk 1]
```diff
@@ -1954,9 +1954,7 @@ describe('TestReportAnalyticsEffects', () => {
           .toHaveBeenCalledTimes(1);
         expect(analyticsProviderMock.logGAEvent)
           .toHaveBeenCalledWith(
-            GoogleAnalyticsEvents.LEGAL_REQUIREMENT,
-            GoogleAnalyticsEventsTitles.ITEM_NAME,
-            legalRequirementsLabels[LegalRequirements.normalStart1],
+            `${GoogleAnalyticsEvents.LEGAL_REQUIREMENT}_NS1`,
             GoogleAnalyticsEventsTitles.ITEM_STATUS,
             'completed',
```
Comment (andrewsetterfield) [inline src/app/pages/test-report/__tests__/test-report.analytics.effects.spec.ts]: use legalRequirementToggleValues.completed please

File: src/app/pages/test-report/__tests__/test-report.analytics.effects.spec.ts [hunk 2]
```diff
@@ -1986,11 +1984,9 @@ describe('TestReportAnalyticsEffects', () => {
           .toHaveBeenCalledTimes(1);
         expect(analyticsProviderMock.logGAEvent)
           .toHaveBeenCalledWith(
-            GoogleAnalyticsEvents.LEGAL_REQUIREMENT,
-            GoogleAnalyticsEventsTitles.ITEM_NAME,
-            legalRequirementsLabels[LegalRequirements.normalStart1],
+            `${GoogleAnalyticsEvents.LEGAL_REQUIREMENT}_NS1`,
             GoogleAnalyticsEventsTitles.ITEM_STATUS,
-            legalRequirementToggleValues.uncompleted
+            'uncompleted',
```
Comment (andrewsetterfield) [inline src/app/pages/test-report/__tests__/test-report.analytics.effects.spec.ts]: restore enum reference please

File: src/app/pages/office/office.analytics.effects.ts [hunk 3]
```diff
@@ -692,21 +692,21 @@ export class OfficeAnalyticsEffects {
...
     concatMap((
-      [, tests, ecoCaptureReason]: [ReturnType<typeof AddEcoCaptureReason>, TestsModel, string, boolean],
+      [, tests]: [ReturnType<typeof AddEcoCaptureReason>, TestsModel, string, boolean],
     ) => {
       // TODO - MES-9495 - remove old analytics
       this.analytics.logEvent(
         formatAnalyticsText(AnalyticsEventCategories.OFFICE, tests),
         formatAnalyticsText(AnalyticsEvents.ECO_CAPTURE_REASON_CHANGED, tests),
-        ecoCaptureReason,
+        'Free text entered',
```
Comment (andrewsetterfield) [inline src/app/pages/office/office.analytics.effects.ts:701]: we have an enum for this

Comment (andrewsetterfield) [reply src/app/pages/office/office.analytics.effects.ts:701]: ah, they're UA.  retracted

**Review by andrewsetterfield — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**